lib/mongoLogs.py

```diff
@@ -0,0 +1,60 @@
+from pymongo import MongoClient
+from bson.objectid import ObjectId
+import os, json, sys
+from configparser import ConfigParser,ExtendedInterpolation
+from datetime import datetime
+import lib.constants as constants
+
+
+root_path = "/opt/sparkjobs/ml-analytics-service/"
```

Comment:
@adithyadinesh0412 is it the server path ? 
are we using this path in anywhere else

Reply:
@rakeshSgr ,
This was discussed in the migrations scripts as well, 
it is similar there, in order to load the constants we will have to provide this path and it remains the same across all the environment.
@Ashwiniev95  can add more on this.

Reply:
@rakeshSgr , It was discussed previously we require to path to be defined